ZenPacks/zenoss/OpenStackInfrastructure/Instance.py

```diff
@@ -47,7 +47,7 @@ def set_host_name(self, hostname):
         if len(hypervisors):
             hypervisor = sorted(hypervisors)[0]
             if len(hypervisors) > 1:
-                LOG.error("Multiple hypervisors were found matching hostname %s - Choosing %s" %
+                LOG.warn("Multiple hypervisors were found matching hostname %s - Choosing %s" %
```

Comment:
Please log a jira with details, i think it needs investigation.  Multiple hypervisors with the same hostname, at least on a standard boring configuration of openstack with libvirt, doesn't seem normal, and if it's happening, we could be losing data.